egret/model_library/transmission/tx_utils.py

```diff
@@ -436,7 +438,12 @@ def _scale_by_baseMVA(normal_op, inverse_op, element, attr_name, attr, baseMVA,
                 _scale_by_baseMVA(normal_op, inverse_op, attr, k, v, baseMVA, attributes)
     elif attr_name in attributes:
         op = _get_op(normal_op, inverse_op, attr_name)
-        element[attr_name] = op( attr , baseMVA )
+        if isinstance(attr, list):
```

Comment:
I am convinced that a blackout will happen in the near future due to failure to apply scaling in some deep, dark corner of an MMS/EMS...